timesketch/lib/utils.py

```diff
@@ -247,9 +247,16 @@ def read_and_validate_jsonl(file_handle):
     # Fields that must be present in each entry of the JSONL file.
     mandatory_fields = ['message', 'datetime', 'timestamp_desc']
     lineno = 0
+    max_length_line_jsonl = current_app.config.get('MAX_LENGTH_LINE_JSONL')
     for line in file_handle:
         lineno += 1
         try:
+            current_len = len(line)
+            if current_len > max_length_line_jsonl:
+                raise RuntimeError(
+                    'Line too long at line {0:n}: {1:s}'.format(
+                        lineno, ','.join(current_len)))
```

Comment:
do we want to raise a runtime error here, or simply log this and move on? I mean if you've got a file with 100k events in them, but 2 lines are too long, don't you want all the rest of the lines to be ingested? and just those two not? and then have a way to notify the user that not everythogn got ingested?

Reply:
Yes, you are right! It's best way.
I modify...